app/views/admin/conferences/edit.html.haml

```diff
@@ -24,5 +24,5 @@
         = f.input :start_hour, :input_html => {size: 2, type: 'number', min: 0, max: 23}
         = f.input :end_hour, :input_html => {size: 2, type: 'number', min: 1, max: 24}
       = f.inputs name: "Registrations" do
-        = f.input :registration_limit, as: :number, in: 0..9999, hint: "Limit the number of registrations to the conference (0 no limit)"
+        = f.input :registration_limit, as: :number, in: 0..9999, hint: "Limit the number of registrations to the conference (0 no limit). You currently have " + pluralize(@conference.registrations.count,'registration','registrations')
```

Comment:
You don't need to add the word in plural. Pluralize can show the proper plural word on its own. You only have to add both singular and plural words, when you want to use something different, eg. 1 person, 2 users.